ui/admin/app/styles/app.scss

```diff
@@ -597,3 +597,68 @@
     padding: 0 sizing.rems(l);
   }
 }
+
+.ordered-series-diagram {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;
+
+  .ordered-series-diagram-item {
+    margin-right: sizing.rems(m);
+    display: grid;
+    grid-template-columns: repeat(auto-fit, 100px);
+    grid-template-rows: repeat(2, 1fr);
+    gap: sizing.rems(xs) 0;
+    grid-template-areas:
+      'icon arrow'
+      'description ...';
+
+    .hds-icon-tile {
+      grid-area: icon;
+      justify-self: center;
+    }
+
+    > span {
+      text-align: center;
+      grid-area: description;
+    }
+
+    .flight-icon-arrow-right {
+      grid-area: arrow;
+      place-self: center;
+    }
+  }
+
+  .ordered-series-diagram-group {
+    border-radius: sizing.rems(xs);
+    display: grid;
+    grid-template-areas:
+      'title title'
+      'primary secondary';
+
+    > span {
+      font-size: sizing.rems(s);
+      color: var(--token-color-foreground-highlight);
```

Comment:
This color should be `--token-color-foreground-highlight-on-surface` to match the wireframe.